Public/Src/App/Bxl/Args.cs

```diff
@@ -371,9 +371,6 @@ public bool TryParse(string[] args, PathTable pathTable, out ICommandLineConfigu
                                     cacheConfiguration.UseDedupStore = true;
                                 }
                             }),
-                        OptionHandlerFactory.CreateBoolOption(
-                            "enableGrpc",

```

Comment:
>enableGrpc [](start = 29, length = 10)

I think you'll need to keep this option around and have it do nothing since there will be builds lingering around that are configured to pass various values to it. Those can and should be cleaned up but making bxl.exe not blow up on the unrecognized argument will make that process easier

Reply:
good catch! 

---
In reply to: [315296455](https://github.com/microsoft/BuildXL/pull/758#discussion_r315296455) [](ancestors = 315296455)